larndsim/detsim_ep.py

```diff
@@ -132,6 +132,10 @@ def z_interval(self, start_point, end_point, x_p, y_p, tolerance, eps=1e-12):
     def erf_hack(self, input):
         return ep.astensor(torch.erf(input.raw))
 
+    def sigmoid(self, x):
+        # Using torch here, otherwise we get overflow -- might be able to get around with a trick
+        return ep.astensor(torch.sigmoid(x.raw))
```

Comment:
Oh no! Another place to remember when de-pytorchizing. Oh well.

Reply:
Yeah, you can avoid this by splitting into x > 0 and x < 0 cases and just writing the function in different ways ([here's a Medium article](https://shaktiwadekar.medium.com/how-to-avoid-numerical-overflow-in-sigmoid-function-numerically-stable-sigmoid-function-5298b14720f6) with terrible screenshots lol), but need to think about how to do this in our case